packages/core/theme/src/components/input.ts

```diff
@@ -236,6 +236,11 @@ const input = tv({
         clearButton: ["transition-opacity", "motion-reduce:transition-none"],
       },
     },
+    isHiddenType: {
+      true: {
+        base: "hidden",
+      },
+    },
```

Comment:
@wingkwong adding this as a variant means that the Input component will receive `isHiddenType` as prop, instead of adding a variant, please use the `data-hidden` prop.

e.g.

`base: "data-[hidden=true]:hidden"`